pkgs/applications/networking/instant-messengers/gomuks/hardcoded_path.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/lib/notification/notify_linux.go b/lib/notification/notify_linux.go
+index f93a95f..b697a11 100644
+--- a/lib/notification/notify_linux.go
++++ b/lib/notification/notify_linux.go
+@@ -32,7 +32,7 @@ func Send(title, text string, critical, sound bool) error {
+ 		if critical {
+ 			soundName = "complete"
+ 		}
+-		exec.Command("paplay", "/usr/share/sounds/freedesktop/stereo/"+soundName+".oga").Run()
++		exec.Command("@pulseaudio@/bin/paplay", "@soundTheme@/share/sounds/freedesktop/stereo/"+soundName+".oga").Run()
+ 	}
+-	return exec.Command("notify-send", args...).Run()
++	return exec.Command("@libnotify@/bin/notify-send", args...).Run()
```

Comment:
If there was only one replacement here, a wrapper script to put notify-send into PATH might be better, to avoid carrying a patch in nixpkgs (?). With two of them, it doesn't matter, I'm just saying this for when upstream fixes the /usr/share path.

Reply:
I expanded the comment near the patch to note this.